.github/workflows/publish-packages.yml

```diff
@@ -111,6 +111,16 @@ jobs:
           GIT_TAG: ${{ inputs.tag }}
           GITHUB_URL: "https://github.com/liveblocks/liveblocks"
         run: |
+          echo "==> Pushing changes to GitHub"
+          if [ "$(git current-branch)" = "main" ]; then
+            if !git push origin main; then
+                echo "WARNING: Could not push this branch to GitHub!" >&2
```

Comment:
NB: I would tend to use the word `ERROR` here instead of `WARNING`. To me from my understanding if we can't push then it means something is not right or somewhat right. So it shouldn't just be marked as warning as we're doing `echo "..." >&2` and we're exiting on `2` after.

I mean from my point of view it would be more meaningful.